lib/ui/prompt.js

```diff
@@ -72,8 +72,9 @@ PromptUI.prototype.processQuestion = function (question) {
 };
 
 PromptUI.prototype.fetchAnswer = function (question) {
+  var screen = this.createScreen();
   var Prompt = this.prompts[question.type];
-  var prompt = new Prompt(question, this.rl, this.answers);
+  var prompt = new Prompt(question, screen, this.rl, this.answers);
```

Comment:
Will take a look at the tests if it is ok. If not I might need some help with an alternative approach.